assignments/function-conversion.js

```diff
@@ -5,26 +5,42 @@
 // };
 // myFunction();
 
+// var myFunction = phrase => phrase
+// myFunction("Function was invoked");
+
+var myFunction = phrase => "Function was invoked";
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+var anotherFunction = example => (param);
+
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
+function add(param1, param2) {
+    return param1 + param2;
+}
+
+var add = (param1, param2) => (param1 + param2);
```

Comment:
var add = (param1, param2) => param1 + param2
you dont need the parenthesis on the end for the return